packages/blast-utilities/run-docker-commands.js

```diff
@@ -54,10 +55,21 @@ const executeNodeMultiCmd = function(arg, enableTty = true) {
   runCommand(dockerComposeCmd + nodeMultiCmd + `'${arg}'`)
 }
 
+const checkDockerStatus = async function() {
+  try {
+    await runCommand(CHECK_DOCKER_STATUS)
+  } catch (error) {
+    if (error instanceof BlastError) {
+      throw new BlastError('Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?')
```

Comment:
do we need the link. to the deamon? Are we sure that it would be the same for windows? We can use only the first part